lib/parse-args.js

```diff
@@ -188,6 +189,28 @@ function buildYargs (withCommands = false) {
   return yargs
 }
 
+function loadConfigFile (path, ext) {
+  let config = {}
+  const jsExts = ['.js', '.cjs']
```

Comment:
What about .mjs? We should likely add a test for this too.

Reply:
While I'd love to see mjs support, the catch there is that ESM doesn't like getting loaded into a CommonJS process. The other way 'round can be done easily, but I think it's out of scope for this ticket to convert this package to ESM... :P

Reply:
@kf6kjg I rolled a patch last night / this morning to include ESM support.  This unfortunately is a big change IMO, and I have to overcome a race condition.  Are there any examples of setting up yargs with async functions?

The good news is that all unit tests were passing and I got 100% code coverage on it.  Take a look [at the diff here](https://github.com/mcknasty/c8/pull/1/commits/fb65ab76ec6c2bedf50856ddf3a11288cdf34179)

Reply:
The advanced docs for yargs shows some: https://github.com/yargs/yargs/blob/main/docs/advanced.md#using-yargs-with-asyncawait

Reply:
@kf6kjg Ok thank you for the link.  I will continue to work on that other patch.  We can mark this conversation as resolved, that ESM is out of scope for this request.  I'll open a feature request for ESM support once this PR is merged.